features/user/account/account_password.feature

```diff
@@ -32,11 +32,13 @@ Feature: User account password change
           And I should see "The entered passwords don't match"
 
     Scenario: Successfully changing my password
-        Given I am on my account password page
-         When I fill in "Current password" with "sylius"
-          And I fill in "New password" with "newpassword"
-          And I fill in "Confirmation" with "newpassword"
-          And I press "Save changes"
+         When I change my account password to "newpassword"
          Then I should be on my account homepage
           And I should see "Your password has been changed successfully!"
 
+    Scenario: Logging in after password change
+        Given I have changed my account password to "newpassword"
+          And I click "Logout"
```

Comment:
Clicking is an action, so `When I click "Logout"` would be more appropriate :wink: